packages/lesswrong/components/sunshineDashboard/ModeratorUserInfo/AltAccountInfo.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import { userGetProfileUrl } from '../../../lib/collections/users/helpers';
+import { useMulti } from '../../../lib/crud/withMulti';
+import { useSingle } from '../../../lib/crud/withSingle';
+import { Link } from '../../../lib/reactRouterWrapper';
+import { registerComponent, Components } from '../../../lib/vulcan-lib';
+import LockIcon from '@material-ui/icons/Lock'
+import LockOpenIcon from '@material-ui/icons/LockOpen'
+
+const styles = (theme: ThemeType): JssStyles => ({
+  root: {
+
+  },
+  icon: {
+    width: 12,
+    height: 12,
+    cursor: "pointer",
+    color: theme.palette.grey[500],
+    marginLeft: 2,
+    marginTop: 1
+  }
+});
+
+export const AltAccountInfo = ({classes, user}: {
+  classes: ClassesType,
+  user: SunshineUsersList
+}) => {
+  const [showAlternateAccounts, setShowAlternateAccounts] = useState<boolean>(false)
+  const { Loading, LWTooltip } = Components
+
+  const { results, loading } = useMulti({
+    terms: {
+      view: "usersByUserIds",
+      userIds: user.associatedClientId?.userIds
+    },
+    collectionName: "Users",
+    fragmentName: 'SunshineUsersList',
+  });
+  
+  const flaggedAccounts = results?.filter(result => result._id !== user._id && (!!result.banned || result.postingDisabled || result.allCommentingDisabled || result.commentingOnOtherUsersDisabled || result.conversationsDisabled || result.karma < 0))
+  const flaggedText = ((flaggedAccounts?.length || 0) >= 1) && <LWTooltip title="One or more users have been banned, had their permissions disabled, or have < 0 karma"><span>
+    ({flaggedAccounts?.length} flagged)
+  </span></LWTooltip>
+
+  return <div className={classes.root}>
+    <em>Alternate accounts detected {flaggedText}</em>
+    <LWTooltip title={<div><p>Click to show alts. Please don't look unless you have reason to suspect this account of suspicious activity</p><p><em>(it's fine for established users to have alts and the mods should not go out of our way to look at them).</em></p></div>}>
+      <span onClick={() => setShowAlternateAccounts(!showAlternateAccounts)}>
+        {showAlternateAccounts ? <LockOpenIcon className={classes.icon}/> : <LockIcon className={classes.icon}/>}
```

Comment:
nbd but I found the lock icon confusing - I would suggest an eye icon instead, to indicate displaying/hiding info

Reply:
The goal was to imply you should not look casually at it. I think an eye conveys "currently hidden" but not "currently hidden and you should consider it somewhat costly to open".

Not sure lock icon is great but I think I'm going to leave it for now.